### ISSUE TITLE
A space between the category page and the main footer when applying specific settings

### ISSUE THREAD

**Issue by wd7080** [magento/magento2#12601]
### Preconditions

1. Magento 2.2.1

### Steps to reproduce

1. Go to Content -> Blocks
2. Add new block
3. Block information:
a. Enable block -> Yes
b. Block title: Contact Us menu
c. Identifier: contactusmenu
d. Store View: All Store Views
e. I am trying to create a contact us form on a cetegory page. I added the following content to the static block editor:
`<p>{{block class="Magento\Contact\Block\ContactForm" template="Magento_Contact::form.phtml"}}</p>`
```
<hr />
<p style="text-align: justify;">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
```
4. Now go to Catalog -> Categories
5. Select the Default Category then -> Click on Add Subcategory
6. Category settings are as follows:
a. Enable Category -> Yes
b. Include in Menu -> Yes
c.  Category name -> Contac Us
d. Content -> Add CMS Block -> Contact Us Menu
e. Display Settings -> Display Mode -> Static block only
f. Display Settings -> Anchor -> No
g. Search engine optimization -> URL Key -> Contact
h. Design -> Use Parent Category Settings -> No
i. Design ->Theme -> Magento Luma
j. Design ->layout -> 1 column
k. Design ->layout update XML -> Empty
l. Design ->apply design to the products -> No
M. Click Save.

### Expected result

1. You can view the contact us page in the menu

### Actual result

1. There is a space between the content and the footer. See the image below.
![magento 2 column](https://user-images.githubusercontent.com/33989344/33772293-bfa9b612-dbf9-11e7-9a4d-3b130835f960.png)

2. The HTML content of this column is:
```
<div class="column main"><input name="form_key" type="hidden" value="sEXSrkYUlSkpqe2v"><div id="authenticationPopup" data-bind="scope:'authenticationPopup'" style="display: none;">
    <script>
        window.authenticationPopup = {"autocomplete":"off","customerRegisterUrl":"http:\/\/domain.com\/customer\/account\/create\/","customerForgotPasswordUrl":"http:\/\/domain.com\/customer\/account\/forgotpassword\/","baseUrl":"http:\/\/domain.com\/"};
    </script>
    <!-- ko template: getTemplate() -->
<!-- /ko -->
</div>
```
3. After the settings I've applied to the category page, this column must be hidden with zero space.
4. Complete image to the page:
![bug](https://user-images.githubusercontent.com/33989344/33772491-66f90102-dbfa-11e7-9694-a6486bc7caa4.png)

**Comment by wd7080** [comment]
@magento-engcom-team were you able to reproduce this? Do you need an update?

**Comment by wd7080** [comment]
Guys, this is really a bug, the main column does not get hidden. Just look into this, please.

**Comment by wd7080** [comment]
Any update on this bug?

**Comment by magento-engcom-team** [comment]
@wd7080, thank you for your report.
We've created internal ticket(s) MAGETWO-85276 to track progress on the issue.

**Comment by sanjay-wagento** [comment]
Hi @magento-engcom-team, @okorshenko
I investigated this issue and working on it.
Please assign to me.
Thanks!

**Comment by magento-team** [comment]
Hi @wd7080. Thank you for your report. 
The issue has been fixed in magento/magento2#13026 by @sanjay-wagento in 2.2-develop branch
Related commit(s):
 - 023fb1a6475cce54493f6b2f0b4277cb6a285773
 - 7727086c752862ada226106c929f1c01ccae8c7a

The fix will be available with the upcoming patch release.

**Comment by okorshenko** [comment]
Hi @wd7080. Thank you for your report. 
The issue has been fixed in magento-engcom/magento2ce#1325 by @magento-engcom-team in 2.3-develop branch
Related commit(s):
 - e84cb3eb79f63f3cac85f5b68b90b82530786d86
 - 742819bdd72880190a992a0f9412af4c0ba417ae
 - cc2b11f47825471e93523db287b4a013da6629d5
 - ae8b1870640227d69e921a5a358fe2addefb5018
 - 35b3c02ca62fa40990fd957c36cd21b7e4f127cd
 - eae893662678539fa602434d92d798ec8a3b3a4c
 - 90e49230db9a94b2b5831c5618b960de35ebc4e0
 - b098dbc4e53efae137286bf056f1e46a2971d52c
 - e8e4cdd29db74ccd13f3078f93c59fb53c4a96d3
 - 058e5e55680e4ca8eb64311f782994fac10a1f74
 - 9368d5da237eb6e158990f2cff8be954242821fe
 - bf19409195b7b93322073026455405ed8820e6af
 - a15ec3421d3d2371429f9c804b91ecd3b6aaf763
 - 9e4ce76a9ad07713d2170420fcd8d237ec776fcb
 - 099568124736c9575bf7498641a0842877260346
 - 6772a2d6658c4677334df0b67159d3a47b6d2155
 - b0d6bfc6599390f31ef3b564fb3e2848037faeef
 - b03735716895f21bee4bc5b5be7d5c7f5a27ba33
 - 8de595178d6dca294d649fde0359a055f07fca69

The fix will be available with the upcoming 2.3.0 release.

**Comment by wd7080** [comment]
Hello
This has not yet been fixed in 2.3.0. I still have the same problem. I really waited for so long for this update to fix this problem. Disappointed!

**Comment by wd7080** [comment]
This issue is never fixed. Any plan for fixing it?

**Comment by magento-engcom-team** [comment]
Hi @wd7080. Thank you for your report. 
The issue has been fixed in magento/magento2#15726 by @sanjay-wagento in 2.3-develop branch
Related commit(s):
 - [ce176d2ab2ae16790f84c7d6836afb51b4d9060a](https://github.com/magento/magento2/commit/ce176d2ab2ae16790f84c7d6836afb51b4d9060a)

The fix will be available with the upcoming 2.3.0 release.

**Comment by sidolov** [comment]
Hi @wd7080. Thank you for your report. 
The issue has been fixed in magento/magento2#15727 by @sanjay-wagento in 2.1-develop branch
Related commit(s):
 - [5e6abd502f500d50cdde91af6c97dfc6f6dc861f](https://github.com/magento/magento2/commit/5e6abd502f500d50cdde91af6c97dfc6f6dc861f)

The fix will be available with the upcoming 2.1.15 release.

**Comment by philgschmidt** [comment]
Still seeing this problem in 2.3.3
What am I missing?
